File: output/xCodeEval_8000/C++/Go/a08055377482990fdb900d1f46aa4fc8.go
package main

import (
	"fmt"
	"sort"
)

const MAGIC = 20000000

var (
	t, la, ma, at int
	to            [1000005]int
	p             [1000005]int
	pp            [1000005]struct {
		first  []int
		second int
	}
	s  = make(map[vector[int]]bool)
	m  [300]map[int]int
	x  [289][289]int

File: output/GPT4o-Mini/xCodeEval_4000/C++/C/a08055377482990fdb900d1f46aa4fc8.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <limits.h>

#define MAGIC 20000000
#define MAX 1000005

int t, la, ma, at, to[MAX];
int p[MAX];
struct {
    int *first;
    int second;
} pp[MAX];
int **s;
int m[300][2];
int x[289][289];
